controllers/controllers.go

```diff
@@ -0,0 +1,21 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/Diegoplas/go-bootcamp-deliverable/app"
+	"github.com/Diegoplas/go-bootcamp-deliverable/useCase"
+	"github.com/gorilla/mux"
+	"github.com/unrolled/render"
+)
+
+func GetPokemon(w http.ResponseWriter, r *http.Request) {
+
+	requestIndex := mux.Vars(r)["pokemon_id"]
+
+	wantedIndex := app.ValidateID(requestIndex)
+
+	response := useCase.GetPokemonFromCSV(wantedIndex)
```

Comment:
This should return an error too, so remember to handle it and return something appropriate

Reply:
Maybe if something went wrong with the csvfile you could return a 500 error, or if no pokemon was found then a 404(Don't use numbers like this, the http package already has all the status codes defined as cosntants)